src/main/java/frc/robot/subsystems/gripper/Gripper.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.subsystems.gripper;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import static frc.robot.Ports.Gripper.*;
+
+public class Gripper extends SubsystemBase {
+    private final VictorSPX motorRight = new VictorSPX(MOTOR);
+    private final VictorSPX motorLeft = new VictorSPX(MOTOR);
+    private final Solenoid solenoid = new Solenoid(SOLENOID);
+
+    private final static Gripper INSTANCE = new Gripper();
+
+    public static Gripper getInstance() {
+        return INSTANCE;
+    }
+
+    /**
+     * Check if motor inverted
+     */
+    private Gripper() {

```

Comment:
use follow, for example motor1.follow(motor2), and then you don't need to set power for both